### PR TITLE
Issue #SB-15055 user sync file provided with proper data

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_and_delete_user from_keyclock/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_and_delete_user from_keyclock/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_and_delete_user from_keyclock/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user_org/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user_org/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user_org/ <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user_org/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/installation/sync_user_org.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+5. When running the script for specified ids, give all the ids as below format in the csv file and no headers required.
+Example:
+3150aa06-333e-4d6d-8d81-3d8e14e7b245
+a354f4c4-a5a7-4eee-9b69-7305cc52a480
+bf1efbd7-4ad6-4e2f-925d-0a6095aeafb0
+54660e04-4d7c-4f12-b307-c5a7174b68d4
+3b87a8a1-0831-453c-8230-b5306daca44f
+b6de8b58-144e-49b2-a65a-835cf81ee700
+
+6. Another way proposed for running failure Id's in the failure log is by running the script
```
Comment (SMYALTAMASH) [inline developer-docs/installation/sync_user_org.md]: Add an instruction to fill the variables like URL and Bearer in the script before running, 
URL: https://DNS
Bearer: value

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://503.qa.docs.sunbird.org/2.3.0/ .<br/> **Note:** Only _2.3.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user/ <br /> - http://503.qa.docs.sunbird.org/2.3.0/developer-docs/installation/sync_user_org/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**